packages/ui-components/Common/DataTag/index.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+
+import styles from './index.module.css';
+
+export type DataTagProps = {
+  kind:
+    | 'event'
+    | 'method'
+    | 'property'
+    | 'class'
+    | 'module'
+    | 'classMethod'
+    | 'ctor';
+  size?: 'lg' | 'md' | 'sm';
+};
+
+// These symbols match up with the types used in
+// node core, and the ones defined at
+// https://github.com/nodejs/api-docs-tooling/blob/main/src/types.d.ts#L22 (`HeadingMetadataEntry['type']`)
+const symbolMap = {
+  event: 'E',
+  method: 'M',
+  property: 'P',
+  class: 'C',
+  module: 'M',
+  classMethod: 'S',
+  ctor: 'C',
+} as const;
+
+export default function DataTag({ kind, size = 'md' }: DataTagProps) {
+  return (
+    <div className={classNames(styles.dataTag, styles[size], styles[kind])}>
+      <span>{symbolMap[kind]}</span>
+    </div>
+  );
+}
```

Comment:
```tsx
const DataTag: FC<DataTagProps> = ({ kind, size = 'md' }) => (
  <div className={classNames(styles.dataTag, styles[size], styles[kind])}>
    <span>{symbolMap[kind]}</span>
  </div>
);

export default DataTag;
```
As a [best practice](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#how-a-new-component-should-look-like-when-freshly-created) you can use arrow function, also no need to return